app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -4656,6 +4657,46 @@ private void RegisterUploadDetails(String ImageName) {
         }
     }
 
+    public void getOfficerVillages(final String OfficerCode) {
+        final ToRestApi toRestApi = new ToRestApi();
+        final JSONObject object = new JSONObject();
+        final String[] str = {""};
+
+
+        Thread thread = new Thread() {
+            public void run() {
+
+                try {
+                    object.put("enrollment_officer_code", OfficerCode);
+                    str[0] = toRestApi.postObjectToRestApiObject(object, "/Locations/GetOfficerVillages");
```

Comment:
Do we have this endpoint in rest api? is this relation used in the mobile app? 
I remember this list was not part of Master Data because is too big and increase the size of the response. 
Can you check if there is any constraints for the EO based on he's location and also if we have this endpoint in REST API?

Reply:
This method is not used in the policies app, github shows it as a modified because i moved it to group all the methods related to master data together. There is also no other code referencing this endpoint.